### DISCUSSION TITLE
sphinx_tabs do not render nicely

### DISCUSSION THREAD

**Discussion by tdegeus** [pradyunsg/furo#482] [Potential Issue] [answered]
I am using a code block in a tab environment:

```rst
.. tabs::

    .. group-tab:: C++

        .. literalinclude:: foo.txt

        Foo:

        1.  Bar::

                foo bar
```
with `sphinx_tabs.tabs`, [see actual docs](https://github.com/tdegeus/GooseFEM/blob/main/docs/examples/statics_linear.rst). The rendering is unfortunately horrible:
[furo.pdf](https://github.com/pradyunsg/furo/files/9127716/furo.pdf)
[see docs](https://goosefem.readthedocs.io/en/latest/examples/statics_linear.html)

**Comment by pradyunsg** [comment, marked as answer]
Yea, sphinx-tabs isn't really supported with this theme.

See https://pradyunsg.me/furo/reference/tabs/ which links to two alternative extensions that serve to replace that. I'm ambivalent on adding support for sphinx-tabs -- it's different enough that I don't feel like adopting it. OTOH, it's reasonably common.

The more pressing issue is that I can't include it in the regular manual testing that I do for this theme since it uses the same directive names as one or both of the other tabs solutions supported.
